CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
-cmake_minimum_required(VERSION 3.10.2)
+cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0091 NEW)
 project(WasmEdge)
 
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
```

Comment:
I would like to know if it's possible to change the `Debug` to `Release` or any other option via a cmake flag. Or is the `Debug` required to fix this issue?

Reply:
There are 4 supported options here:

MultiThreaded
MultiThreadedDLL
MultiThreadedDebug
MultiThreadedDebugDLL

The fix is to use non-DLL versions

`MultiThreaded$<$<CONFIG:Debug>:Debug>` means
`MultiThreadedDebug` for `Debug` configuration
`MultiThreaded` otherwise